package.json

```diff
@@ -95,6 +96,9 @@
     },
     "localforage": {
       "optional": true
+    },
+    "localforage-removeitems": {
```

Comment:
this peer dependency is optional, similar to `localforage`, reference https://github.com/Expensify/react-native-onyx/pull/132/files#r873998291